src/node/in_memory.rs

```diff
@@ -427,7 +425,8 @@ impl<S: std::fmt::Debug + ForkSource> InMemoryNodeInner<S> {
 
         if let Some(ref mut eip712_meta) = request_with_gas_per_pubdata_overridden.eip712_meta {
             if eip712_meta.gas_per_pubdata == U256::zero() {
-                eip712_meta.gas_per_pubdata = MAX_GAS_PER_PUBDATA_BYTE.into();
+                eip712_meta.gas_per_pubdata =
+                    get_max_gas_per_pubdata_byte(VmVersion::latest()).into();
```

Comment:
Just in case, it wont work for old bootloaders (not sure if these are supported in external node anyway)

Reply:
I actually realized that we override this value in old VM versions, so it is fine